collector.go

```diff
@@ -206,3 +225,57 @@ func getEventTotalData(c *eventTotalCache, db *sql.DB) ([]eventTotal, error) {
 
 	return etList, nil
 }
+
+func defaultDagRunStates() map[string]float64 {
+	return map[string]float64{
+		"success": 0.0,
+		"failed":  0.0,
+		"running": 0.0,
+	}
+}
+
+func getDagRunStateData(db *sql.DB) ([]dagRunState, error) {
+
+	preparedStmt := "SELECT COUNT(*), COALESCE(dag_id, ''), COALESCE(state, '') FROM dag_run GROUP BY dag_id, state"
+
+	stmt, err := db.Prepare(preparedStmt)
```

Comment:
A prepared statement isn't really needed here since no parameters are passed... You could directly use something like:

    rows, err := db.Query("SELECT COUNT(*), COALESCE(dag_id, ''), COALESCE(state, '') FROM dag_run GROUP BY dag_id, state")